conf/knnaas_config.yaml

```diff
@@ -0,0 +1,16 @@
+# Config file for KNN as a Service
+
+
+# Port number on which to run the service. (Required)
+port: 9090
+
```

Comment:
Can port number be more exotic ? 9090 seems to be used for some other services. But not famous ones though.